rust-escrow/stable-escrow/src/storage.rs

```diff
@@ -0,0 +1,35 @@
+use near_sdk::{
+    borsh::{self, BorshDeserialize, BorshSerialize},
+    collections::UnorderedMap,
+    near_bindgen,
+    serde::{Deserialize, Serialize},
+    AccountId, Balance,
+};
+
+#[near_bindgen]
+#[derive(BorshDeserialize, BorshSerialize)]
+pub struct ConditionalEscrow {
+    pub deposits: UnorderedMap<AccountId, Balance>,
+    pub total_funds: Balance,
+    pub metadata: Metadata,
+}
+
+#[derive(BorshSerialize, BorshDeserialize, Serialize, Deserialize, Clone)]
+pub struct Metadata {
+    pub expires_at: u64,
+    pub funding_amount_limit: u128,
+    pub unpaid_funding_amount: u128,
+    pub nep_141_account_id: AccountId,
```

Comment:
@susguzman should we set this to USN or USDT by default? Without letting the users set the value from the UI?

What would you choose, USN or USDT? and why

Reply:
It's better if we keep it as a parameter so the contract can work with any NEP141.